packages/react-components/react-avatar/src/components/AvatarGroupOverflow/AvatarGroupOverflow.types.ts

```diff
@@ -1,16 +1,62 @@
+import * as React from 'react';
+import type { AvatarSizes } from '../Avatar/Avatar.types';
+import type { AvatarGroupProps } from '../AvatarGroup/AvatarGroup.types';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import type { PopoverProps, PopoverSurface } from '@fluentui/react-popover';
+import type { TooltipProps } from '@fluentui/react-tooltip';
 
 export type AvatarGroupOverflowSlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<PopoverProps>>;
+
+  /**
+   * Overflow indicator that triggers the Popover.
+   */
+  overflowButton: NonNullable<Slot<'button'>>;
+
+  /**
+   * List that contains the overflow AvatarGroupItems.
+   */
+  overflowContent: NonNullable<Slot<'div'>>;
+
+  /**
+   * PopoverSurface that contains the overflow content.
+   */
+  overflowSurface: NonNullable<Slot<typeof PopoverSurface>>;
```

Comment:
Ditto, 'overflow' is redundant in these names. Perhaps `popoverContent` and `popoverSurface`?